Userland/Libraries/LibImageDecoderClient/Client.cpp

```diff
@@ -33,8 +31,7 @@ Optional<DecodedImage> Client::decode_image_with_known_path(DeprecatedString con
     auto encoded_buffer = encoded_buffer_or_error.release_value();
 
     memcpy(encoded_buffer.data<void>(), encoded_data.data(), encoded_data.size());
-    auto response_or_error = try_decode_image_with_known_path(path, move(encoded_buffer));
-
+    auto response_or_error = try_decode_image(move(encoded_buffer), mime_type.value());
```

Comment:
This doesn't seem right, `mime_type` is not guaranteed to have a value? Should be just:

```suggestion
    auto response_or_error = try_decode_image(move(encoded_buffer), mime_type);
```

(or `move()`d, not sure if `try_decode_image` takes by value or const ref)